docs/excel/run-code-on-document-open.md

```diff
@@ -40,13 +42,12 @@ let behavior = await Office.addin.getStartupBehavior();
 
 ## How to run code when the document opens
 
-When your add-in is configured to load on document open, it will run immediately. The `Office.initialize` event handler will be called. Place your startup code in the `Office.initialize` event handler.
-
-The following code shows how to register an event handler for change events from the active worksheet. If you configure your add-in to load on document open, this code will register the event handler when the document is opened. You can handle change events before the task pane is opened.
+When your add-in is configured to load on document open, it will run immediately. The `Office.initialize` event handler will be called. Place your startup code in the `Office.initialize` or `Office.onReady` event handler.
 
+The following Excel add-in code shows how to register an event handler for change events from the active worksheet. If you configure your add-in to load on document open, this code will register the event handler when the document is opened. You can handle change events before the task pane is opened.
 
 ```JavaScript
-//This is called as soon as the document opens.
+// This is called as soon as the document opens.
 //Put your startup code here.
```

Comment:
You didn't change this line, but would be great if we can add a space after the `//`! 
```suggestion
// Put your startup code here.
```